api/middleware/logger.go

```diff
@@ -0,0 +1,123 @@
+package middleware
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	opentracing "github.com/opentracing/opentracing-go"
+	log "github.com/sirupsen/logrus"
+	jaeger "github.com/uber/jaeger-client-go"
+	macaron "gopkg.in/macaron.v1"
+)
+
+var (
+	logHeaders = false
```

Comment:
what's with this variable? should this be configurable?

Reply:
perhaps in the future, disabled by default for now.